vital/io/mesh_io.cxx

```diff
@@ -104,6 +105,13 @@ read_ply2(const std::string& filename)
 {
   check_input_file(filename);
   std::ifstream input_stream(filename.c_str());
+
+  if ( ! input_stream.is_open() )
```

Comment:
It looks like a very similar check is already done in the `check_input_file` function: https://github.com/Kitware/kwiver/blob/75369be0a22ff9b719ce216dd05125fd4f14328e/vital/io/mesh_io.cxx#L69

Reply:
Thanks, I've removed the redundant checks as you suggested. The commit has been amended, and the changes are now pushed.